draft-packages/badge/KaizenDraft/Badge/Badge.tsx

```diff
@@ -3,27 +3,40 @@ import classNames from "classnames"
 
 import styles from "./styles.module.scss"
 
-type Variant = "default" | "active" | "dark"
+type Variant = "default" | "active" | "dark" | "dot"
+type Size = "small" | "large"
 
 export interface BadgeProps {
-  readonly children: string
+  readonly children?: string
+  /**
+   * The "dark" variant is no longer in the UI kit
+   */
   readonly variant?: Variant
+  /**
+   * renders reversed colors. Use on purple background
+   */
   readonly reversed?: boolean
+  /**
+   * Supports "small" and "large" sizes - defaults to "small"
+   */
+  readonly size?: Size
```

Comment:
Just a comment, also as someone not super familiar with TS best practices, but unless the `Size` and `Variant` types are used again somewhere, would it not be easier to inline them for easier readability?
Example:
```
readonly size?: "small" | "large"
```
This way it's directly clear what `size` is without having to scan back up for its definition.